addons/l10n_pe_pos/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
I think you should ideally include the "Part of Odoo" header in all Python files.